draft-trammell-taps-post-sockets.md

```diff
@@ -285,14 +286,22 @@ there is long-term state associated with it (via the underlying Association;
 see {{association}}), and it may be able to reactivated, but messages cannot
 be sent and received immediately.
 
-If supported by the underlying transport protocol stack, a Carrier may
-be forked: creating a new Carrier associated with a new
-Carrier at the same remote endpoint. The semantics of the usage of multiple Carriers based on the same Association are application-specific. When a Carrier is forked, its corresponding Carrier at the remote
-endpoint receives a fork request, which it must accept in order to fully
-establish the new carrier. Multiple Carriers between endpoints are
-implemented differently by different transport protocol stacks, either using
-multiple separate transport-layer connections, or using multiple streams of
-multistreaming transport protocols.
+Carriers may actively be made dormant and disposed of, or may become dormant
+because the underlying transport protocol stack determines that an underlying
+connection has been lost and there is insufficient state in the Association to
+re-establish it (e.g., in the case of a server-side Carrier where the client's
+address has changed unexpectedly). Passive close can be handled by the
+application via an event on the carrier.
```

Comment:
I thought we decided that carriers cannot become dormant. Was this an oversight from a previous version of this PR?

Reply:
A carrier cannot be made dormant and resurrected. But if a PSI goes away and an endpoint has enough state to seamlessly re-establish it, why would we force the application to worry about that?

Reply:
You just keep the carrier. It's still active. It just does have a transient. Correct, why would an application need o know about this!

Reply:
If there's no transient, it can't be active. If you're an active opener and you still have enough remote information for re-establishment, then this can happen seamlessly. If you're a passive opener then you're out of luck, and will need the event.

"via an event on the carrier" always means it's a thing you can ignore. Don't care? Don't listen. But IMO for events like this saying we *always* hide it from the application isn't the right thing to do.